vaadin-combo-box-flow/src/main/resources/META-INF/resources/frontend/comboBoxConnector.js

```diff
@@ -1,370 +1,376 @@
+                }
+                if ( typeof combo.$validate == 'undefined'){
+                    combo.$validate = combo.validate;
+                    combo.validate = function() {
+                        return !(comboBox.focusElement.invalid = comboBox.invalid);
+                    };
+                }
+            })
+
+            const disableTextFieldClientValidation =  tryCatchWrapper(function (field, comboBox){
+                if ( typeof field.$checkValidity == 'undefined'){
+                    field.$checkValidity = field.checkValidity;
+                    field.checkValidity = function() { return !comboBox.invalid; };
+                }
+            })
+
+            const enableTextFieldClientValidation = tryCatchWrapper(function (field){
+                if ( field.$checkValidity ){
+                    field.checkValidity = field.$checkValidity;
+                    delete field.$checkValidity;
+                }
+            })
+
+            const enableClientValidation = tryCatchWrapper(function (combo){
+                if ( combo.$checkValidity ){
+                    combo.checkValidity = combo.$checkValidity;
+                    delete combo.$checkValidity;
+                }
+                if ( combo.$validate ){
+                    combo.validate = combo.$validate;
+                    delete combo.$validate;
+                }
+            })
+
+            const commitPage = tryCatchWrapper(function (page, callback) {
+                let data = cache[page];
+
+                if (comboBox._clientSideFilter) {
+                    performClientSideFilter(data, callback)
+
+                } else {
+                    // Remove the data if server-side filtering, but keep it for client-side
+                    // filtering
+                    delete cache[page];
+
+                    // FIXME: It may be that we ought to provide data.length instead of
+                    // comboBox.size and remove updateSize function.
+                    callback(data, comboBox.size);
+                }
+            })
+
+            // Perform filter on client side (here) using the items from specified page
+            // and submitting the filtered items to specified callback.
+            // The filter used is the one from combobox, not the lastFilter stored since
+            // that may not reflect user's input.
+            const performClientSideFilter = tryCatchWrapper(function (page, callback) {
+
+                let filteredItems = page;
+
+                if (comboBox.filter) {
+                    filteredItems = page.filter(item =>
+                        comboBox.$connector.filter(item, comboBox.filter));
+                }
+
+                callback(filteredItems, filteredItems.length);
+            })
+
+            // https://github.com/vaadin/vaadin-combo-box-flow/issues/232
+            comboBox.addEventListener('opened-changed', e =>
```

Comment:
I think these two event listeners (`opened-changed` and `custom-value-set`) could also be wrapped with `tryCatchWrapper` even if just for consistency and to be more future proof in case more code is added to these handlers later. Not a blocker though.

Reply:
Fixed.